src/lib/utils/options/readers/package-json.ts

```diff
@@ -0,0 +1,84 @@
+import { dirname, join, resolve } from "path";
+import * as fs from "fs";
+
+import type { OptionsReader } from "..";
+import type { Logger } from "../../loggers";
+import type { Options } from "../options";
+import { ok } from "assert";
+import { nicePath } from "../../paths";
+
+export class PackageJsonReader implements OptionsReader {
+    // Should run after the TypeDoc config reader but before the TS config
+    // reader, so that it can still specify a path to a `tsconfig.json` file.
+    priority = 150;
+
+    name = "package-json";
+
+    read(container: Options, logger: Logger): void {
+        const cfgPath = container.getValue("options");
+        const absPath = resolve(cfgPath);
+
+        // Search for `package.json` first at the provided config path, or at
+        // `cfgPath/package.json` if the provided path is a directory.
+        const path = [absPath, join(absPath, "package.json")].find(
+            (p) => fs.existsSync(p) && fs.statSync(p).isFile()
+        );
+
+        if (!path) {
+            if (container.isSet("options")) {
```

Comment:
This shouldn't be an error - options specified the location of typedoc.json, for test projects I frequently have several typedoc.json files. I think we should just silently do nothing here.

Reply:
Gotcha, we'll carry on silently if a file path can't be resolved.